fvm/src/syscalls/vm.rs

```diff
@@ -14,11 +14,12 @@ pub enum Never {}
 unsafe impl SyscallSafe for Never {}
 
 // NOTE: this won't clobber the last syscall error because it directly returns a "trap".
-pub fn abort(
+pub fn exit(
     context: Context<'_, impl Kernel>,
     code: u32,
     message_off: u32,
     message_len: u32,
+    blk: u32,
```

Comment:
nit: let's switch the ordering. It's cleaner if the optional error message goes last.

Reply:
ok.

Reply:
done.